crates/plugins/mediator-coordination/src/model/stateful/entity.rs

```diff
@@ -28,7 +28,7 @@ pub struct RoutedMessage {
     #[serde(rename = "_id")]
     #[serde(skip_serializing_if = "Option::is_none")]
     pub id: Option<ObjectId>,
-    pub message: Message,
+    pub message: Vec<String>,
     pub recipient_did: String,
```

Comment:
if you want to store multiple entries, you have to convert the field name to its plural form.